app/forms/instance-create.tsx

```diff
@@ -774,21 +771,25 @@ const AdvancedAccordion = ({
                 <MiniTable.HeadCell className="w-12" />
               </MiniTable.Header>
               <MiniTable.Body>
-                {attachedFloatingIpsData.map((item, index) => (
-                  <MiniTable.Row
-                    tabIndex={0}
-                    aria-rowindex={index + 1}
-                    aria-label={`Name: ${item.name}, IP: ${item.ip}`}
-                    key={item.name}
-                  >
-                    <MiniTable.Cell>{item.name}</MiniTable.Cell>
-                    <MiniTable.Cell>{item.ip}</MiniTable.Cell>
-                    <MiniTable.RemoveCell
-                      onClick={() => detachFloatingIp(item.name)}
-                      label={`remove floating IP ${item.name}`}
-                    />
-                  </MiniTable.Row>
-                ))}
+                {attachedFloatingIpsData.map((item, index) =>
+                  item ? (
+                    <MiniTable.Row
+                      tabIndex={0}
+                      aria-rowindex={index + 1}
+                      aria-label={`Name: ${item.name}, IP: ${item.ip}`}
+                      key={item.name}
+                    >
+                      <MiniTable.Cell>{item.name}</MiniTable.Cell>
+                      <MiniTable.Cell>{item.ip}</MiniTable.Cell>
+                      <MiniTable.RemoveCell
+                        onClick={() => detachFloatingIp(item.name)}
+                        label={`remove floating IP ${item.name}`}
+                      />
+                    </MiniTable.Row>
+                  ) : (
+                    <></>
+                  )
+                )}
```

Comment:
This change is unnecessary because the type of `attachedFloatingIpsData` is `FloatingIp[]` — they can't be falsy. I suspect your VS Code had a blip where it was using TS 5.4, for which this filter fails to narrow the type properly.

```ts
const attachedFloatingIpsData = attachedFloatingIps
  .map((ip) => attachableFloatingIps.find((fip) => fip.name === ip.floatingIp))
  .filter((ip) => !!ip)
```